database/crdb.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: crdb
+  title: GNU/Linux wireless regulation database
```

Comment:
If we'll be super-pendantic, this is not *GNU*-specific, but actually Linux kernel-specific.

Reply:
Fixed. Thanks.